src/api/profiles.ts

```diff
@@ -77,6 +77,8 @@ export async function parseProfileMember(
 			collections: member.collection !== undefined,
 			inventory: member.inventory?.inv_contents !== undefined,
 			vault: member.inventory?.personal_vault_contents !== undefined,
+			essence: member.currencies?.essence !== undefined,
```

Comment:
Do we need an `essence` API flag? The idea of these were to match the in-game toggles

Reply:
oh yeah true, i found out that essence api gets turned off if you have inventory api disabled, but we could just go off of that too

Reply:
I think we should stick with these toggles, matching the ones in game. So remove the essence one here and the PR looks good otherwise (besides conflicts but yeah)